src/components/DetailSidebar/styles.ts

```diff
@@ -3,11 +3,12 @@ import ListItemText from '@material-ui/core/ListItemText';
 
 import ListItem from '../../muiComponents/ListItem';
 
-export const TitleListItem = styled(ListItem)({
-  paddingLeft: 0,
-  paddingRight: 0,
-  paddingBottom: 0,
-});
+export const TitleListItem = styled(ListItem)(`
+padding: 0 0.5rem;
```

Comment:
Why are you using `rem` ? The rem unit means "The root element's font-size".

Reply:
Yea, wanted to make it relative with its roots! Should I change it to px?

Reply:
There is no need for styling now. We have to remove the hover effect (grey background color). you can remove button={true}. Please see the review summary.

Reply:
You want me to remove hover styling and `button={true}` from both the component?

Reply:
yeah, I think the button hover effect was a mistake from previous migrations. Actually mine.

Reply:
ok will remove it by next week, a bit busy with my college exams !

Reply:
done 👍

Reply:
@anikethsaha again the snapshots :(

Reply:
I did update the snapshots after removing button prop